### DISCUSSION TITLE
How to build and install `ninja-python-distributions` use `pip`?

### DISCUSSION THREAD

**Discussion by chuanwise** [scikit-build/scikit-build#1110] [Q&A] [answered]
I want to install `ninja` package in my anaconda python environment, but no tag matched in the official builds, so I want to build it from source. I clone and saw <https://github.com/scikit-build/ninja-python-distributions/blob/master/docs/make_a_release.rst>, but pypi package setup script like `setup.py` is not found so that I don't know how to build and install it to current environment. Does anyone can help me?

**Comment by henryiii** [comment, marked as answer]
`pip install -v .` would do it. Though it's better to use the conda package for ninja unless you really need the PyPI distribution for something.
